lib/src/crypto_subtle.dart

```diff
@@ -334,23 +334,23 @@ extension type JSRsaOtherPrimesInfo(JSObject _) implements JSObject {
 
 TypedData getRandomValues(TypedData array) {
   if (array is Uint8List) {
-    window.crypto.getRandomValues(array.toJS);
-    return array;
+    final values = window.crypto.getRandomValues(array.toJS);
+    return (values as JSUint8Array).toDart;
```

Comment:
Is there any need for this cast, doesn't `getRandomValues` always return the first argument that it is given?